src/main/java/ToDoApplication/DAO/DAO.java

```diff
@@ -1,4 +1,4 @@
 package ToDoApplication.DAO;
 
-public interface DAO {
+public interface Dao {
```

Comment:
Does it make sense to remove this interface? Since it doesn't have any method. What do you think?